simulaqron/tests/auto/quick/merges/configs/Alice.cfg

```diff
@@ -6,6 +6,6 @@
 # [name], [hostname], [port number]
 #
 
-Alice, localhost, 8811
+# Alice, localhost, 8811
 
```

Comment:
To be deleted?
There might be more of those scattered around in the tests

Reply:
It is used by the test, the old config files are still supported